### PR TITLE
JAVA-3576 - Align module names, folder names and artifact id

### PR DESCRIPTION
**Pull request by freelansam**

### CODE REVIEW

File: core-java-modules/core-java-jpms/decoupling-pattern1/consumermodule/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <dependencies>
         <dependency>
             <groupId>com.baeldung.servicemodule</groupId>
-            <artifactId>servicemodule1</artifactId>
+            <artifactId>servicemodule</artifactId>
```
Comment (lor6) [inline core-java-modules/core-java-jpms/decoupling-pattern1/consumermodule/pom.xml:20]: should this be consumermodule?

File: core-java-modules/core-java-jpms/decoupling-pattern2/providermodule/pom.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <dependencies>
         <dependency>
             <groupId>com.baeldung.servicemodule</groupId>
-            <artifactId>servicemodule2</artifactId>
+            <artifactId>servicemodule</artifactId>
```
Comment (lor6) [inline core-java-modules/core-java-jpms/decoupling-pattern2/providermodule/pom.xml:20]: should this be providermodule?

File: core-java-modules/core-java-jpms/decoupling-pattern1/consumermodule/pom.xml (hunk 1 above)
Comment (freelansam) [reply core-java-modules/core-java-jpms/decoupling-pattern1/consumermodule/pom.xml:20]: @lor6 - the artifactId for the module is actually consumermodule, line 20 is adding a dependency on another module, hence it is servicemodule.

File: core-java-modules/core-java-jpms/decoupling-pattern2/providermodule/pom.xml (hunk 2 above)
Comment (freelansam) [reply core-java-modules/core-java-jpms/decoupling-pattern2/providermodule/pom.xml:20]: The same reason as above applies here as well i think.

**Pull request merged**